up-l3/utwin/v2/README.adoc

```diff
@@ -111,6 +111,16 @@ A Topic is a cache entry that can be refreshed individually. Refreshing cache in
 
 uTwin instances act as a distributed networks of caches that can communicate to retrieve values without requesting a new publication, in case a uTwin instance is lacking a Topic entry in its own cache, by retrieving the last event from the cache (uTwin instance) running on the same device as the publisher of the topic). Using the analogy of a cache, it may also be useful to implement, later, a flush() command, that would remove the last known state for specific Topics, or all Topics of a given uDevice, etc.
 
+=== uTwin Cache Management
+
+uTwin will initialize its cache on startup with the following scenarios:
+
+* when running without persistent storage, the cache will be initialized to an empty condition
+* when running with persistent storage (future option), the cache can be initialized to the last saved state for each Topic.
+
+uTwin MUST indicate when its capacity is exhausted and it is unable to accept a new / updated SetLastValue command.
+
+uTwin should actively manage its cache to ensure that the oldest and lowest priority Topics are removed from the cache in order to ensure that new/updated content can continue to be accepted.  This should be done as a background/idle task to decouple it from impacting active uTwin transactions, potentially based upon a set threshold (e.g. 95% full).  Criteria for protection of higher priority content will need to be established, along with determination of "staleness" for specific topics/topic classes.
```

Comment:
I really don't think cache requirements are generic and per the protocol but should be documented in the implementations of uTwin.  Also "oldest" would need to be elaborated on, same for lowest priority. If we want to provided detailed caching rules then this needs to be elaborated a lot more.

Reply:
Added refinements to the text, and also for "oldest", as well as priority.   I can add further details including a sequence diagram if this would be of value.